grails-app/conf/Config.groovy

```diff
@@ -128,7 +128,7 @@ openId {
 }
 
 featureToggles {
-    // someFeature = <boolean>
+    filterSource = 'database' // Todo - DN: This can go back to being a boolean when we've removed loading filters from the DB
```

Comment:
As I understand it no_db will still leave us with a boolean option, @danfruehauf. We wanted to be able to load from GeoServer of from XML files to allow more options for developing/troubleshooting. So we'll need a switch between those options.